test-fixtures/urn/f15/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "description": "F15: MET-573. Similar to F14? Should this have a 020?",
```

Comment:
I think test is distinct enough because of $y

Reply:
Consult MH. We decided that $5 FI-Vapaa is the best indicator of free depositness.